packages/blade/src/components/Menu/VisualSubComponents/MenuDivider.web.tsx

```diff
@@ -0,0 +1,11 @@
+import { getDividerMarginTokens } from '../tokens';
+import type { StyledPropsBlade } from '~components/Box/styledProps';
+import { Divider } from '~components/Divider';
+import { useTheme } from '~utils';
+
+const MenuDivider = (styledProps: StyledPropsBlade): React.ReactElement => {
```

Comment:
Why do we need a MenuDivider component? Can we use the Divider component directly?

Reply:
No we have to add exact negative margin for end-to-end divider